dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -0,0 +1,80 @@
+import z from 'zod';
+import { ObjectId, UpdateFilter, WithoutId, WithId, FindOneAndUpdateOptions, Collection } from 'mongodb';
+import { TfmFacilitySchema } from './tfm-facility.schema';
+import { mongoDbClient } from '../../drivers/db-client';
+
+export type RawTfmFacility = WithId<object>;
+
+export type ParsedTfmFacility = z.infer<typeof TfmFacilitySchema>;
+
+export class TfmFacilitiesRepo {
+  /**
+   * Gets the TFM facilities collection
+   * @returns The TFM facilities collection
+   */
+  public static async getCollection(): Promise<Collection<WithoutId<RawTfmFacility>>> {
```

Comment:
Do we want to move away from this being `public`? I assume it would be a real pain to make this private